src/main/java/com/redhat/pantheon/model/module/Module.java

```diff
@@ -139,7 +139,7 @@ public ModuleRevision createNextRevision() {
         }
 
         private String generateNextRevisionName() {
-            return "v" + stream(this.getChildren()).collect(counting());
+            return "" + (stream(this.getChildren()).collect(counting()) + 1);
```

Comment:
I see you've changed the revision naming strategy. No complaints from me. There might be a few header comments (in the Module class I think) where we might want to update the examples.